contracts/StrategyCurve3crv.sol

```diff
@@ -117,104 +219,69 @@ abstract contract StrategyCurveBase is BaseStrategy {
         }
     }
 
-    // fire sale, get rid of it all!
+    /** 
+    @notice Unstake LP tokens from gauge and return the total LP token balance.
+    This function is used during emergency exit instead of `prepareReturn()` to
+    liquidate all of the Strategy's positions back to the Vault.
+    @return uint256 LP token balance in the strategy
+    */
     function liquidateAllPositions() internal override returns (uint256) {
         uint256 _stakedBal = stakedBalance();
         if (_stakedBal > 0) {
-            // don't bother withdrawing zero
             gauge.withdraw(_stakedBal);
         }
         return balanceOfWant();
     }
 
+    /** 
+    @notice Do anything necessary to prepare this Strategy for migration. In this
+    case, only unstaking LP tokens from the gauge is required.
+    @param _newStrategy New strategy to which we are migrating. Not used in this
+    strategy.
+    */
     function prepareMigration(address _newStrategy) internal override {
         uint256 _stakedBal = stakedBalance();
         if (_stakedBal > 0) {
             gauge.withdraw(_stakedBal);
         }
-    }
 
-    function protectedTokens()
-        internal
-        view
-        override
-        returns (address[] memory)
-    {}
+        want.safeTransfer(_newStrategy, want.balanceOf(address(this)));
 
-    /* ========== SETTERS ========== */
-
-    // These functions are useful for setting parameters of the strategy that may need to be adjusted.
-
-    // Set the amount of CRV to be locked in Yearn's veCRV voter from each harvest. Default is 10%.
-    function setKeepCRV(uint256 _keepCRV) external onlyAuthorized {
-        require(_keepCRV <= 10_000);
-        keepCRV = _keepCRV;
-    }
-
-    // This allows us to manually harvest with our keeper as needed
-    function setForceHarvestTriggerOnce(bool _forceHarvestTriggerOnce)
-        external
-        onlyAuthorized
-    {
-        forceHarvestTriggerOnce = _forceHarvestTriggerOnce;
+        // Migrate the strategy's balances of CRV and GNO, if any.
```

Comment:
this is a material change in how we prepare the strategy for a migration. Because we no longer swap the tokens within harvest, when we migrate we need the old strategy to send any balance of the rewards tokens CRV and GNO to the new strategy.

Reply:
This is not wrong but in the past we never cared about claiming profit.
If we left profit in an old strategy, we have sweep. 
You can leave this logic, no biggie.

Reply:
good point about sweep. I guess when dealing with a migration we can always run a transfer to the new strategy from governance if there were rewards left behind. I think I'll leave it for GC, but I understand how in mainnet gas would argue otherwise.